active_model_serializers.gemspec

```diff
@@ -57,4 +57,10 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'grape', ['>= 0.13', '< 1.0']
   spec.add_development_dependency 'json_schema'
   spec.add_development_dependency 'rake', ['>= 10.0', '< 12.0']
+
+  spec.post_install_message = <<-EOF
+NOTE: The default key case for the JsonApi adapter has changed to dashed.
+See https://github.com/rails-api/active_model_serializers/blob/master/docs/general/key_transform.md
+for more information on configuring this behavior.
```

Comment:
Champion, eh?! I guess I have a vested interest now :laughing: 

The only time that this issue really affects people is when the options is nil, where it falls back to the default for the adapter. I would recommend that instead of accepting `nil` silently, we issue a deprecation notice when the option is not set.